src/WebJobs.Extensions.DocumentDB/Services/IDocumentDBService.cs

```diff
@@ -4,13 +4,14 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
 
 namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
 {
     /// <summary>
     /// An abstraction layer for communicating with a DocumentDB account.
     /// </summary>
-    public interface IDocumentDBService
+    internal interface IDocumentDBService
```

Comment:
Please add a test to PublicSurfaceTests for this binding, and review your surface area